azurelinuxagent/common/utils/processutil.py

```diff
@@ -16,16 +16,27 @@
 #
 # Requires Python 2.6+ and Openssl 1.0+
 #
-import os
-import signal
-from errno import ESRCH
 
-from azurelinuxagent.common.exception import ExtensionError
 from azurelinuxagent.common.future import ustr
 
 TELEMETRY_MESSAGE_MAX_LEN = 3200
 
 
+def read_output(stdout, stderr):
```

Comment:
Since now this is an independent public function, it should have its own unit tests (can be done on later PR)